docs/examples/pytorch/resnet50/main.py

```diff
@@ -13,6 +13,7 @@
 import torch.utils.data
```

Comment:
This would deserve a separate PR

Reply:
This file is just for testing, I will remove it from PR.